src/event-bus/EventStore.ts

```diff
@@ -0,0 +1,130 @@
+import { Event, PersistedEvent } from '../shared/events';
```

Comment:
This filename uses camelCasing, which is not used elsewhere in the project. Let's stay consistent.